Makefile

```diff
@@ -113,7 +134,82 @@ preload-images:
 
 validate: pkg/subctl/operator/common/embeddedyamls/yamls.go
 
-.PHONY: test validate build ci clean generate-clientset generate-embeddedyamls generate-operator-api operator-image preload-images
+all: manager
+
+# Run tests
+#test: generate fmt vet manifests
+	#go test ./... -coverprofile cover.out
+
+# Build manager binary
+manager: generate fmt vet
+# We check BUILD_ARGS since that's what the compile script uses
+ifeq (--debug,$(findstring --debug,$(BUILD_ARGS)))
+	go build -o bin/submariner-operator main.go
+else
+	go build -ldflags -s -w -o bin/submariner-operator main.go
+endif
+
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: generate fmt vet manifests
+	go run ./main.go
+
+# Install CRDs into a cluster
+install: manifests
+	$(KUSTOMIZE) build config/crd | sed 's/$$(VERSION)/$(VERSION)/g' | kubectl apply -f -
+
+# Uninstall CRDs from a cluster
+uninstall: manifests
+	$(KUSTOMIZE) build config/crd | sed 's/$$(VERSION)/$(VERSION)/g' | kubectl delete -f -
+
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: manifests clusters preload-images
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/default | sed 's/$$(VERSION)/$(VERSION)/g' | kubectl apply -f -
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests:
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+# Run go fmt against code
+fmt:
+	go fmt ./...
+
+# Run go vet against code
+vet:
+	go vet ./...
+
```

Comment:
If we want those targets they should live in shipyard.

Reply:
I think we can remove `fmt` and `vet` as we already have what @dfarrell07 prepared as part of shipyard
`make manifests` actually generates the manifests files CRDS, RBAC, etc... should I move it to shipyard as well?